requirements-static.txt

```diff
@@ -1,2 +1,3 @@
-ruff==0.14.0
+git+https://github.com/Earth-Information-System/fireatlas.git
```

Comment:
It'd be better to point to a specific release, or failing that at least a specific commit, so that it won't change on us unexpectedly.